command/rubycommand/rubycommand.go

```diff
@@ -213,3 +215,57 @@ func IsGemInstalled(gem, version string) (bool, error) {
 
 	return findGemInList(out, gem, version)
 }
+
+// IsSelectedRbenvRubyInstalled checks if the selected ruby version is installed via rbenv.
+// Ruby version is set by
+// 1. The RBENV_VERSION environment variable
+// 2. The first .ruby-version file found by searching the directory of the script you are executing and each of its
+// parent directories until reaching the root of your filesystem.
+// 3.The first .ruby-version file found by searching the current working directory and each of its parent directories
+// until reaching the root of your filesystem.
+// 4. The global ~/.rbenv/version file. You can modify this file using the rbenv global command.
+func IsSelectedRbenvRubyInstalled(workdir string) (bool, string, error) {
+	absWorkdir, err := pathutil.AbsPath(workdir)
+	if err != nil {
+		return false, "", fmt.Errorf("Failed to expand (%s), error: %s", workdir, err)
```

Comment:
Error message specific to one possible underlying error.

The returned error should clarify the reason for the failure (e.g. problem during expansion) and the outer should only refer to the purpose of the call itself (e.g. `could not get absolute path for workfir`)

Also, please use lowercase initial letter, because you can't be sure if this error will be the topmost error message in the calling code.

Same goes for all error messages below.